tests/backup/backup_basic_test.go

```diff
@@ -142,6 +147,7 @@ var _ = BeforeSuite(func() {
 	for _, provider := range providers {
 		switch provider {
 		case drivers.ProviderAws:
+			log.Infof(" The driver is aws")
```

Comment:
Do you really need this it looks like a debug log line?

Reply:
removed